bindings/rust/bench/benches/handshake.rs

```diff
@@ -43,17 +48,27 @@ pub fn bench_handshake_params(c: &mut Criterion) {
         for ec_group in [SECP256R1, X25519] {
             let mut bench_group =
                 c.benchmark_group(format!("handshake-{:?}-{:?}", handshake_type, ec_group));
-            bench_handshake_for_library::<S2NHarness>(&mut bench_group, handshake_type, ec_group);
-            bench_handshake_for_library::<RustlsHarness>(
-                &mut bench_group,
-                handshake_type,
-                ec_group,
-            );
-            bench_handshake_for_library::<OpenSslHarness>(
+            bench_handshake_for_library::<S2NHarness>(
                 &mut bench_group,
+                "s2n-tls",
                 handshake_type,
                 ec_group,
             );
+            #[cfg(not(feature = "s2n-only"))]
+            {
+                bench_handshake_for_library::<RustlsHarness>(
+                    &mut bench_group,
+                    "rustls",
+                    handshake_type,
+                    ec_group,
+                );
+                bench_handshake_for_library::<OpenSslHarness>(
+                    &mut bench_group,
+                    "openssl",
+                    handshake_type,
+                    ec_group,
+                );
+            }
```

Comment:
Talked offline, decided that keeping the feature was ultimately worth it because historical performance builds/runs faster without compiling the rustls/openssl benches. However, we'll rename the feature to `historical-perf` to more accurately reflect the purpose of the feature and call out what it does in the readme.